### PR TITLE
HBASE-22768 Revert to MPIR 2.9

### PR DESCRIPTION
**Pull request by petersomogyi**

### CODE REVIEW

File: src/site/site.xml [hunk 1]
```diff
@@ -84,19 +84,19 @@
...
       <item name="Other resources" href="resources.html"/>
     </menu>
     <menu name="Project Information">
-      <item name="Project Summary" href="summary.html"/>
+      <item name="Project Summary" href="project-summary.html"/>
       <item name="Dependency Information" href="dependency-info.html"/>
-      <item name="Source Repository" href="scm.html"/>
-      <item name="Issue Tracking" href="issue-management.html"/>
+      <item name="Source Repository" href="source-repository.html"/>
+      <item name="Issue Tracking" href="issue-tracking.html"/>
```
Comment (saintstack) [inline src/site/site.xml]: Why these changes? The site works with them in place?

File: pom.xml [hunk 2]
```diff
@@ -3728,12 +3729,11 @@
               <report>dependency-info</report>
               <report>dependency-management</report>
               <report>index</report>
-              <report>issue-management</report>
-              <report>licenses</report>
-              <report>mailing-lists</report>
+              <report>issue-tracking</report>
+              <report>mailing-list</report>
```
Comment (saintstack) [inline pom.xml]: Did report names change? No licenses?

File: pom.xml [hunk 3]
```diff
@@ -3720,6 +3720,7 @@
     <plugins>
       <plugin>
         <artifactId>maven-project-info-reports-plugin</artifactId>
+        <version>2.9</version>
```
Comment (saintstack) [inline pom.xml]: I suppose we make defines for these usually but if one-time use, I think this fine in the pom.

File: src/site/site.xml (hunk 1 above)
Comment (petersomogyi) [reply src/site/site.xml]: These files are generated by Maven Project Info Report tool and from 3.0 the filenames are changed. With the rollback to version 2.9 the filenames need to be changed back as well.

File: pom.xml (hunk 2 above)
Comment (petersomogyi) [reply pom.xml]: Yes, some reports were renamed similarly to the generated html files.
I removed `licenses` because we don't use license.html any more but have a link to https://www.apache.org/licenses/ which is an ASF site requirement. This improvement was done by @symat in HBASE-20405.

File: pom.xml (hunk 3 above)
Comment (petersomogyi) [reply pom.xml]: Good point! Let me change this to a property.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 44 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| -1 | mvninstall | 104 | root in master failed. |
| +1 | compile | 172 | master passed |
| +1 | mvnsite | 1085 | master passed |
| +1 | shadedjars | 282 | branch has no errors when building our shaded downstream artifacts. |
| -1 | javadoc | 86 | root in master failed. |
||| _ Patch Compile Tests _ |
| -1 | mvninstall | 85 | root in the patch failed. |
| +1 | compile | 170 | the patch passed |
| +1 | javac | 170 | the patch passed |
| +1 | mvnsite | 914 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 2 | The patch has no ill-formed XML file. |
| +1 | shadedjars | 267 | patch has no errors when building our shaded downstream artifacts. |
| -1 | hadoopcheck | 72 | The patch causes 10 errors with Hadoop v2.8.5. |
| -1 | hadoopcheck | 161 | The patch causes 10 errors with Hadoop v2.9.2. |
| -1 | hadoopcheck | 258 | The patch causes 10 errors with Hadoop v3.1.2. |
| -1 | javadoc | 92 | root in the patch failed. |
||| _ Other Tests _ |
| +1 | unit | 13189 | root in the patch passed. |
| +1 | asflicense | 43 | The patch does not generate ASF License warnings. |
| | | 16852 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/433 |
| Optional Tests | dupname asflicense javac javadoc unit shadedjars hadoopcheck xml compile mvnsite |
| uname | Linux 834774c22740 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-433/out/precommit/personality/provided.sh |
| git revision | master / 586e177ded |
| Default Java | 1.8.0_181 |
| mvninstall | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/4/artifact/out/branch-mvninstall-root.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/4/artifact/out/branch-javadoc-root.txt |
| mvninstall | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/4/artifact/out/patch-mvninstall-root.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/4/artifact/out/patch-javac-2.8.5.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/4/artifact/out/patch-javac-2.9.2.txt |
| hadoopcheck | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/4/artifact/out/patch-javac-3.1.2.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/4/artifact/out/patch-javadoc-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/4/testReport/ |
| Max. process+thread count | 5658 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/4/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by petersomogyi** [comment]
Errors are strange, it looks like the local maven repository is broken. A similar problem happened at #419. Retriggering build.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 118 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 370 | master passed |
| +1 | compile | 196 | master passed |
| +1 | mvnsite | 1280 | master passed |
| +1 | shadedjars | 301 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 184 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 319 | the patch passed |
| +1 | compile | 190 | the patch passed |
| +1 | javac | 190 | the patch passed |
| +1 | mvnsite | 1224 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 3 | The patch has no ill-formed XML file. |
| +1 | shadedjars | 298 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1001 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| -1 | javadoc | 84 | root in the patch failed. |
||| _ Other Tests _ |
| -1 | unit | 16750 | root in the patch failed. |
| +1 | asflicense | 39 | The patch does not generate ASF License warnings. |
| | | 22762 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.master.TestMasterShutdown |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=18.09.7 Server=18.09.7 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/8/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/433 |
| Optional Tests | dupname asflicense javac javadoc unit shadedjars hadoopcheck xml compile mvnsite |
| uname | Linux 316e4d582d32 4.15.0-52-generic #56-Ubuntu SMP Tue Jun 4 22:49:08 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-433/out/precommit/personality/provided.sh |
| git revision | master / caa05356a1 |
| Default Java | 1.8.0_181 |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/8/artifact/out/patch-javadoc-root.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/8/artifact/out/patch-unit-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/8/testReport/ |
| Max. process+thread count | 4580 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/8/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 68 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 333 | master passed |
| +1 | compile | 194 | master passed |
| +1 | mvnsite | 1280 | master passed |
| +1 | shadedjars | 299 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 181 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 325 | the patch passed |
| +1 | compile | 194 | the patch passed |
| +1 | javac | 194 | the patch passed |
| +1 | mvnsite | 1218 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 3 | The patch has no ill-formed XML file. |
| +1 | shadedjars | 299 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1034 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 178 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 16618 | root in the patch failed. |
| +1 | asflicense | 29 | The patch does not generate ASF License warnings. |
| | | 22648 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=18.09.7 Server=18.09.7 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/9/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/433 |
| Optional Tests | dupname asflicense javac javadoc unit shadedjars hadoopcheck xml compile mvnsite |
| uname | Linux 9cc98b845c9d 4.15.0-52-generic #56-Ubuntu SMP Tue Jun 4 22:49:08 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-433@2/out/precommit/personality/provided.sh |
| git revision | master / 24b970ec51 |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/9/artifact/out/patch-unit-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/9/testReport/ |
| Max. process+thread count | 4543 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-433/9/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by petersomogyi** [comment]
2 test failed but those are unrelated to this change.
